content/runbooks/kubernetes/CPUThrottlingHigh.md

```diff
@@ -0,0 +1,47 @@
+---
+title: CPU Throttling High
```

Comment:
Where does this `CPU Throttling High` content come from?

Reply:
part of it comes from [upstream prs](https://github.com/prometheus-operator/runbooks/pulls) that has not been merged.
the [upstream repo](https://github.com/prometheus-operator/runbooks) currently only provides runbooks with just over 30 rules, but kubesphere has more than 100 built-in alert rules and we have to do the rest.